src/integration-test/integration/AggregationsClient.test.ts

```diff
@@ -0,0 +1,365 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised";
+import { expect, use } from "chai";
+import "reflect-metadata";
+import { accessToken, testIModel, testIModelGroup } from "../utils";
+import { AggregationPropertyCreate, AggregationPropertyType, AggregationPropertyUpdate, AggregationsClient, AggregationTableCreate, AggregationTableSetCreate, AggregationTableSetUpdate, AggregationTableUpdate, MappingCreate, MappingsClient } from "../../reporting";
+use(chaiAsPromised);
+
+describe("Aggregations Client", () => {
+  const aggregationsClient: AggregationsClient = new AggregationsClient();
```

Comment:
Initialize clients in `utils/GlobalSetup.ts` so that they would respect the configured base url.

Reply:
First you will need to merge the main branch to your branch to have the latest changes.